index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <section id="welcome-section">
+        <h1>Lorem Ipsum</h1>
+    </section>
+    <nav id="navbar" ><a href="#welcome_section"></a></nav>
+    <div id="projects">
```

Comment:
## @MarsIncarnate, it would be amazing if you could add text or image objects within your '#projects' section to show your '.project-tile' and link elements.